packages/tailwindcss/src/compile.ts

```diff
@@ -51,7 +48,9 @@ export function compileCandidates(
         // variants used.
         let variantOrder = 0n
         for (let variant of candidate.variants) {
-          variantOrder |= 1n << BigInt(variants.indexOf(variant))
+          let order = variantOrderMap.get(variant)
+          if (order === undefined) continue
```

Comment:
We can use `!` instead of the if check because the variants inside candidate.variants are guaranteed to be present in getVariantOrder().